.github/workflows/render-readme.yaml

```
@@ -40,15 +40,25 @@ jobs:
 
       - name: Render README
         env:
-          GITHUB_TOKEN: ${{ steps.app-token.outputs.token }}
+          GITHUB_TOKEN: "${{ steps.app-token.outputs.token }}"
         shell: bash
         run: python ./.github/scripts/render-readme.py
 
+      # below is yoinked from https://gist.github.com/swinton/03e84635b45c78353b1f71e41007fc7c
       - name: Commit Changes
-        shell: bash
+        env:
+          GITHUB_TOKEN: "${{ steps.app-token.outputs.token }}"
+          FILE_TO_COMMIT: "README.md"
+          DESTINATION_BRANCH: "main"
         run: |
-          git config --global user.name "bot-ross"
-          git config --global user.email "98030736+bot-ross[bot]@users.noreply.github.com"
-          git add ./README.md
-          git commit -m "Render README.md" || echo "No changes to commit"
-          git push origin || echo "No changes to push"
+          export TODAY=$( date -u '+%Y-%m-%d' )
+          export MESSAGE="chore: regenerate $FILE_TO_COMMIT for $TODAY"
```

```suggestion
          export MESSAGE="chore: regenerate $FILE_TO_COMMIT for $TODAY [skip ci]"
```
Otherwise you will end up with an endless loop